library/systemd/src/modules/Systemd.rb

```diff
@@ -52,7 +52,7 @@ def Installed
     # Check whether systemd init is currently running
     # @return boolean true if systemd init is running
     def Running
-      FileUtils.IsDirectory(@systemd_mountdir) == true
+      SCR.Read(path(".target.string"), "/proc/1/comm")&.chomp == "systemd"
```

Comment:
Another alternative:
```suggestion
      File.directory?(File.join(@running_dir, "/system"))
```

Reply:
I am not sure if we should call it in target as I think systemd cannot run in chroot and mount bind proc, so it is in the end also local proc

Reply:
That is why shouldn't matter, but yep, it is better to use local